x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout.tsx

```diff
@@ -78,11 +90,15 @@ const Cards = ({ data }: { data: Card[] }) => (
   <EuiFlexGrid direction="column" gutterSize={'l'}>
     {data.map((card) => (
       <EuiFlexItem key={card.title} style={{ display: 'block' }}>
-        <EuiCard textAlign="left" title={card.title}>
+        <EuiCard textAlign="left" title={card.title} hasBorder>
           <EuiDescriptionList
             compressed={false}
             type="column"
             listItems={card.listItems.map((v) => ({ title: v[0], description: v[1] }))}
+            style={{ flexFlow: 'column' }}
```

Comment:
make sure both 96 and 98 are needed. might work with just one of them

Reply:
yeah, it doesn't add it 😄